doc/source/modules/gui/plot/statswidget.rst

```diff
@@ -32,18 +32,18 @@
    :members:
 
 
-:class:`BasicGridStatsWidget` class
+:class:`_BasicGridStatsWidget` class
```

Comment:
Why putting private classes in the documentation?

Reply:
come from a class rename, BasicGridStatsWidget -> _BasicGridStatsWidget